packages/relay/index.js

```diff
@@ -7,14 +7,15 @@ import {
   type RefetchRelayProp,
   type PaginationRelayProp,
   commitLocalUpdate as _commitLocalUpdate,
+  type Environment,
 } from '@kiwicom/relay';
 import { Alert } from '@kiwicom/mobile-localization';
 
-import PublicEnvironment from './src/PublicEnvironment';
-import PrivateEnvironment from './src/PrivateEnvironment';
 import ConnectionManager from './src/ConnectionManager';
 import type { QueryRendererProps as _QueryRendererProps } from './types';
 
+export { default as PublicEnvironment } from './src/PublicEnvironment';
```

Comment:
So we don't really need the PrivateEnvironment?

Reply:
We don't need to export at this point at least